ingest_graph_validator/hydrators/ingest_hydrator.py

```diff
@@ -120,10 +120,9 @@ def get_edges(self):
                             if relationship_name == 'DERIVED_BY_PROCESSES':
                                 edges.append(Relationship(end_node, 'DUMMY_EXPERIMENTAL_DESIGN', start_node))
 
+                            self._logger.debug(f"({start_node['id']})-[:{relationship_name}]->({end_node['id']})")
                         except KeyError:
-                            self._logger.debug(f"Missing end node at a [{start_node['label']}] entity.")
-
-                        self._logger.debug(f"({start_node['id']})-[:{relationship_name}]->({end_node['id']})")
+                            self._logger.debug(f"Missing end node at a [{start_node['id']}] entity.")
```

Comment:
shouldn't this be raised? What's the implication of hvaing a missing end node?

Reply:
I guess it's okay not to raise it if it'll fail in the graph validation tests. It'll just complete building the graph with the given entities in the submission/spreadsheet first.

Reply:
The graph validator completes its run and creates the available entities and links. But I'm not sure how the wrangler would get to know about the missing things, as at the moment, only `logger.info` gets printed to console